docs-src/docs/reference/setup-cli.md

```diff
@@ -1,278 +1,212 @@
 
 ### Usage
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoPadding">**l0-setup plan** _prefixName_ _terraformArguments_</div>
-  </div>
-</div>
-
-###Required parameters
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">_prefixName_</div>
-    <div class="divCell">The name of a Layer0Prefix for which you want to plan an update.</div>
-  </div>
-</div>
-
-###Optional arguments
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">_terraformArguments_</div>
-    <div class="divCell">The terraform arguments to pass to Layer0. Arguments passed in this way must follow the syntax specified in the [Terraform Commands (CLI) Documentation](https://www.terraform.io/docs/commands/index.html).</div>
-  </div>
-</div>
+```
+$ l0-setup endpoint [options] <instance_name> 
+```
+
+### Options
+* `-i, --insecure` - Show environment variables that allow for insecure settings
+* `-d, --dev` - Show environment variables that are required for local development
+* `-s --syntax` - Choose the syntax to display environment variables 
+(choices: `bash`, `cmd`, `powershell`) (default: `bash`)
 
 ---
+## Destroy
+The **destroy** command is used to destroy all resources associated with a Layer0 instance.
 
-##Restore
-The **restore** command is used to restore Layer0 configuration files that were previously backed up to an S3 bucket using the [**backup**](#backup) command.
+!!! warning "Caution"
+	Destroying a Layer0 instance cannot be undone; if you created backups of your Layer0 configuration using the **push** command, those backups will also be deleted when you run the **destroy** command.
 
 ### Usage
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoPadding">**l0-setup restore** [--access\_key=_awsAccessKeyID_] [--secret\_key=_awsSecretAccessKeyID_] [--region=_awsRegion_] _prefixName_</div>
-  </div>
-</div>
-
-###Required parameters
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">_prefixName_</div>
-    <div class="divCell">The name of the Layer0Prefix that you want to restore.</div>
-  </div>
-</div>
-
-###Optional arguments
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">--access\_key=_awsAccessKeyID_</div>
-    <div class="divCell">The Access Key ID of an IAM user associated with the AWS stack in which the Layer0 instance was created.</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--secret\_key=_awsSecretAccessKeyID_</div>
-    <div class="divCell">The Secret Access Key ID of an IAM user associated with the AWS stack in which the Layer0 instance was created.</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--region=_awsRegion_</div>
-    <div class="divCell">The AWS region in which the Layer0 instance resides.</div>
-  </div>
-</div>
+```
+$ l0-setup destroy [options] <instance_name> 
+```
 
----
+### Options
+* `--force` - Skips confirmation prompt
```

Comment:
keeping since flag already exists in older versions, also it will likely take on more behavior than just prompting in the future